src/Compilers/CSharp/Portable/Symbols/SymbolExtensions.cs

```diff
@@ -115,16 +115,14 @@ internal static NamespaceOrTypeSymbol ContainingNamespaceOrType(this Symbol symb
             return null;
         }
 
-        internal static Symbol ContainingNonLambdaMember(this Symbol containingMember)
+        internal static Symbol ContainingNonLambdaMember(this Symbol member)
```

Comment:
Please revert.

---
In reply to: [91636283](https://github.com/dotnet/roslyn/pull/15720#discussion_r91636283) [](ancestors = 91636283)